libraries/rush-lib/assets/rush-init/common/config/rush/rush-alerts.json

```diff
@@ -63,6 +63,27 @@
        */
       /*[LINE "HYPOTHETICAL"]*/ "endTime": "2024-01-05",
 
+      /**
+       * (OPTIONAL) Specifies the maximum frequency at which this alert can be displayed within a defined time period.
+       * Options are:
+       *  "always" (default) - no limit on display frequency,
+       *  "monthly" - display up to once per month
+       *  "weekly" - display up to once per week
+       *  "daily" - display up to once per day
+       *  "hourly" - display up to once per hour
+       */
+      /*[LINE "HYPOTHETICAL"]*/ "maximumDisplayInterval": "always",
+
+      /**
+       * (OPTIONAL) Determines the order in which this alert is shown relative to other alerts, based on urgency.
+       * Options are:
+       *  "high" - displayed first
+       *  "mid" - moderate urgency
+       *  "normal" (default) - standard urgency
+       *  "low" - least urgency
```

Comment:
(It's not obvious whether `normal` should be higher or lower than `mid`.)

You would need at least simultaneously 3 active alerts for these distinctions to matter, so I'm thinking that `high`, `normal`, and `low` are probably enough.  Basically "high" means we want to make sure people see it first, whereas "low" means it's not important, and "normal" is everything else.  We can easily add more levels in the future if it's really necessary.